packages/fetchai/skills/ml_train/handlers.py

```diff
@@ -201,7 +201,7 @@ def _handle_data(self, ml_trade_msg: MlTradeMessage) -> None:
             )
             self.context.task_manager.enqueue_task(
```

Comment:
get your task_id, save it locally, then use it in behaviour to check if task is done